app/jobs/socure_docv_results_job.rb

```diff
@@ -15,14 +15,36 @@ def perform(document_capture_session_uuid:, async: true, docv_transaction_token_
       document_capture_session
 
     timer = JobHelpers::Timer.new
-    response = timer.time('vendor_request') do
+    client_response = timer.time('vendor_request') do
       socure_document_verification_result
     end
+
+    document_response_validator = Idv::DocumentResponseValidator.new(
```

Comment:
in the api image_upload_form, the document_response_validator is only used if the form_response is successful ... here we are creating a form_response solely for the sake of this constructor ... do we need to pass in the form_response into the constructor? 🤔